app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">FoodApp</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
```

Comment:
Instead of leaving a TODO comment, this is covered by checking for MVP readiness in #1 .